scipy-1.0/paper.tex

```diff
@@ -88,15 +88,17 @@ \subsection*{Test and benchmark suite}
     confirming that performance generally remains constant or improves over the commit hash history of the project allows us to objectively
     measure that our code base is improving, to empower scientific applications.
 
-    Consider the following asv benchmark results, spanning roughly nine years of project history, that demonstrate the gradual performance
-    improvements in \texttt{scipy.spatial.cKDTree.query()} (nearest-neighbor search) performance, using command:
+    Consider the following asv benchmark results, spanning roughly nine years of project history. These demonstrate the gradual performance
+    improvements in a nearest-neighbor search, \texttt{scipy.spatial.cKDTree.query()} using the command:
 
     \texttt{python run.py run -e -s 800 --bench "\textbackslash btime\_query\textbackslash b" "02de46a546..b3ddb2c"}
+    
+   Any pull request can be compared against the \texttt{master} branch with the command \texttt{asv continuous master new-feature}. This will provide a benchmark against the master branch and the branch a new feature is implemented in. More features are available in the documentation, including arguments to select which benchmarks to run.
```

Comment:
Perhaps move this paragraph so that it doesn't break up the discussion of project history profiling? Could go before the paragraph that starts with `Consider...`

Reply:
Ah, I see. I added a `\ref` command, which is what I normally use. When this document is compiled, it will produce "Figure 2" or something (and hyperlink the reference if we compile with that package).